app/Filament/Chantier/Resources/Chantiers/Tables/ChantiersTable.php

```
@@ -136,18 +149,20 @@ public static function configure(Table $table): Table
                             $quote = $record->quote;
-                            if (!$quote) return;
+                            if (! $quote) {
+                                return;
+                            }
 
                             $percentage = $data['percentage'] / 100;
                             $amountHt = $quote->total_ht * $percentage;
                             $amountTva = $quote->total_tva * $percentage;
 
-                            $invoice = \App\Models\Commerce\CustomerInvoice::create([
+                            $invoice = CustomerInvoice::create([
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$invoice'. (undefined)

(UnusedLocalVariable)